### ISSUE TITLE
Never tries to re-init replication on failure.

### ISSUE THREAD

**Issue by mjbnz** [nikaro/ansible-role-postgresql-bdr#3]
If any of the tasks following `PostgreSQL BDR | Enable extensions` fails, the role never tries again.

A few things in play here:
- once extension is enabled, config is never tried again. perhaps check for active replication first ?
- presence of sql files in /tmp stops `PostgreSQL BDR | Copy upstream SQL script` from re-copying. remove `*script.changed` from `when` ?

**Comment by nikaro** [comment]
> once extension is enabled, config is never tried again. perhaps check for active replication first ?

I agree that this would be the correct way of doing it. Unfortunately i do not use BDR anymore (i use repmgr now) and the gears of BDR are not fresh anymore in my mind, so i would not know how to fix it. Feel free to submit a PR.
